recommender/models/service.py

```diff
@@ -35,3 +35,13 @@ class Service(MarketplaceDocument):
     required_services = ListField(ReferenceField("Service"))
     dataframe = BinaryField(blank=True)
     tensor = ListField(FloatField(), blank=True)
+
+    meta = {
+        "indexes": [
+            {
+                "fields": ["$name", "$description", "$tagline"],
+                "default_language": "english",
+                "weights": {"title": 10, "content": 5, "tagline": 2},
+            }
+        ]
+    }
```

Comment:
Are those all indexes? Maybe order_type/country should also be here.

Reply:
order_type and country are fixed values and they are included in filters, not in text_search